spec/features/admin_cannot_modify_users_data_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+RSpec.describe 'Admin', type: :feature do
+  create_admin
+  create_user
+  feature 'admin' do
+    it 'cannot modify users account data' do
+      visit login_path
+
+      fill_in 'Username', with: 'admin'
+      fill_in 'Password', with: 'admin'
+
+      click_button "Login"
+
+      expect(page).to have_content('Admin Dashboard')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.